src/HomePage/Heading/Heading.scss

```diff
@@ -1,6 +1,16 @@
-.heading {
-  font-family: 'Questrial', 'Lato';
-  text-align: center;
-  font-weight: 300;
-  font-size: 70px;
+.header {
+  display: flex;
+  align-items: center;
+  min-height: 100vh;
```

Comment:
This seems to be pushing it a bit further below the bottom of the screen, probably because of the navbar. You'd want to make this 100vh - height of the navbar

Reply:
Added in!